### PR TITLE
Simple application with WebSocket

### PR DESCRIPTION
**Pull request by ehsavoie**
Updating the original application with a display of the returned message...
Put some style into a CSS instead of directly into the page for easier maintenance.

### CODE REVIEW

**Comment by sgilda** [comment]
You improved this a lot! thanks!

I ran into a few  strange problems when testing this. I think the connection timed out, the status changed to Closed, but when I sent the message, I got the response:

"The message was sent! You can view the message in the server log."

Another time I got:
Error! undefined

I'll look at it closer next week. Thanks again!

**Comment by sgilda** [comment]
I issued a pull back to @ehsavoie with updates to the README file and CLI scripts to modify the server configuration:

File: websocket-hello/README.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
+jboss-websocket-hello: A simple WebSocket application
+========================
+Author: Sande Gilda  
+Level: Beginner  
+Technologies: CDI, JSF, WebSocket  
+Summary: An simple example of a WebSocket application  
+Target Product: EAP  
+Product Versions: EAP 6.1, EAP 6.2, EAP 6.3
```
Comment (maschmid) [inline websocket-hello/README.md:8]: Websockets are new in EAP 6.3

File: websocket-hello/README.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+* A JavaScript enabled WebSocket HTML client.
+* A WebSocket server endpoint that uses annotations to interact with the WebSocket events.
+* A `jboss-web.xml` file configured to enable WebSockets
+
+The JBoss EAP WebSocket implementation is available via the `jbossweb-<VERSION>.jar` file.
+
+System requirements
+-------------------
+
+The application this project produces is designed to be run on Red Hat JBoss Enterprise Application Platform 6.1 or later.
```
Comment (maschmid) [inline websocket-hello/README.md:25]: Websockets are new in EAP 6.3

File: websocket-hello/src/main/webapp/index.html [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+-->
+<!-- Plain HTML page that kicks us into the app -->
+
+<html>
+<head>
+<meta http-equiv="Refresh" content="0; URL=hello.jsf">
```
Comment (maschmid) [inline websocket-hello/src/main/webapp/index.html:21]: hello.xhtml doesn't really use JSF, perhaps all of index.xhtml should be just put into this index.html file and not use JSF at all?

File: websocket-hello/README.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
+jboss-websocket-hello: A simple WebSocket application
+========================
+Author: Sande Gilda  
+Level: Beginner  
+Technologies: CDI, JSF, WebSocket
```
Comment (maschmid) [inline websocket-hello/README.md:5]: it doesn't use CDI, nor JSF really

**Comment by sgilda** [comment]
Replaced this pull with: https://github.com/jboss-developer/jboss-eap-quickstarts/pull/941

**Pull request closed without merging**